frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -0,0 +1,183 @@
+    <Form onFinish={onFinish} form={form} initialValues={defaultValues} layout="vertical">
+      <Row style={{ display: "grid", marginBottom: 16 }}>
+        <Alert
+          message="Please note that this feature is experimental. Bounding boxes are expected to have sizes that are multiple of 20×20×18 voxel."
+          type="warning"
+          showIcon
+        />
+      </Row>
+      <Row gutter={8}>
+        <Col span={24}>
+          <FormItem
+            hasFeedback
+            name="modelName"
+            label="Model Name"
+            rules={[
+              {
+                required: true,
+                message: "Please name the model that should be trained.",
+              },
+            ]}
+          >
+            <Input autoFocus />
+          </FormItem>
+        </Col>
+      </Row>
+      <FormItem
+        hasFeedback
+        name="modelCategory"
+        label="Model Category"
+        rules={[
+          {
+            required: true,
+            message: "Please select a model category.",
+          },
+        ]}
+      >
+        <Select>
+          <Select.Option value="em_neurons">EM Neurons</Select.Option>
+          <Select.Option value="em_nuclei">EM Nuclei</Select.Option>
+        </Select>
+      </FormItem>
+      <FormItem
+        hasFeedback
+        name="imageDataLayer"
+        label="Image Data Layer"
+        hidden={colorLayers.length === 1}
+        rules={[
+          {
+            required: true,
+            message: "Please select a layer whose image data should be used for training.",
+          },
+        ]}
+      >
+        <LayerSelection layers={colorLayers} tracing={tracing} style={{ width: "100%" }} />
+      </FormItem>
+      <LayerSelectionFormItem
+        chooseSegmentationLayer
+        layers={segmentationLayers}
+        fixedLayerName={fixedSelectedLayer?.name}
+        tracing={tracing}
+        label="Groundtruth Layer"
+      />
+      <Row gutter={8}>
+        <Col span={24}>
+          <FormItem hasFeedback name="comment" label="Comment">
+            <Input />
+          </FormItem>
+        </Col>
+      </Row>
+      <Collapse
+        style={{ marginBottom: 8 }}
+        items={[
+          {
+            key: "advanced",
+            label: "Advanced",
+            children: (
+              <FormItem name="workflowYaml" label="Workflow Description (yaml)">
+                <TextArea
+                  className="input-monospace"
+                  autoSize={{
+                    minRows: 6,
+                  }}
+                  style={{
+                    fontFamily: 'Monaco, Consolas, "Lucida Console", "Courier New", monospace',
+                  }}
+                />
+              </FormItem>
+            ),
+          },
+        ]}
+        defaultActiveKey={[]}
+      />
+
+      <FormItem hasFeedback name="dummy" label="Training Data">
+        <div>
+          {userBoundingBoxes.length} bounding boxes ({bboxesVoxelCount / 1000000} MVx)
```

Comment:
Maybe the `({bboxesVoxelCount / 1000000} MVx)` part can be done without a constant conversion factor? E.g. use the format utils/ implement a rudimentary version for vx like done with `formatBytes`.